### ISSUE TITLE
Is this bug?

### ISSUE THREAD

**Issue by NikitaMishin** [status-im/vyper-debug#18]
_If this is a bug report, please fill in the following sections.
If this is a feature request, delete and describe what you would like with examples._

## What was wrong?
 vyper-run  name_registry.vy "register(0x12,0x000000000000000f000001000000000000000001)"
produce me assertion failed  when == and !=. I succesfully call this method from remix
![Screenshot_2019-05-06_18-40-00](https://user-images.githubusercontent.com/22191229/57236742-66fa5880-702e-11e9-844f-b643c9627726.png)

### Code that produced the error

```py

registry: map(bytes[100], address)

@public
def register(name: bytes[100], owner: address)->address:
    assert self.registry[name] == ZERO_ADDRESS 
    self.registry[name] = owner
    return owner


@public
@constant
def lookup(name: bytes[100]) -> address:
    return self.registry[name]
....
with call 
 vyper-run  name_registry.vy "register(0x12,0x000000000000000f000001000000000000000001)"
CODE_TO_REPRODUCE 
```

### Full error output 

```sh
ERROR_HERE 
```

### Expected Result

_This section may be deleted if the expectation is "don't crash"._

```sh
EXPECTED_RESULT
``` 

### Environment

```sh
# run this:
$ python -m eth_utils

# then copy the output here:
OUTPUT_HERE
``` 

## How can it be fixed?

Fill this section in if you know how this could or should be fixed.

**Comment by NikitaMishin** [comment]
it's like assert checks post factum

**Comment by NikitaMishin** [comment]
About __init__, there is problem vyper-run. There no name for __init__ function (i think it was removed):

    for func in abi:
        try:
            func_signature = func['name']
        except KeyError:
            # constructor and fallback functions don't have a name
            continue
This defenitely error.

**Comment by NikitaMishin** [comment]
Find bug with assert! There was extra call (unused)

**Comment by jacqueswww** [comment]
Fixed in #20, thanks! :)

**Comment by NikitaMishin** [comment]
Will you update version to get it latest version through pip install?
